ocs_ci/cleanup/aws/defaults.py

```diff
@@ -2,7 +2,10 @@
 Defaults module for AWS cleanup
 """
 AWS_REGION = 'us-east-2'
-CLUSTER_PREFIXES_TO_EXCLUDE_FROM_DELETION = {'jnk': '36'}
-MINIMUM_CLUSTER_RUNNING_TIME_FOR_DELETION = 10
+CLUSTER_PREFIXES_SPECIAL_RULES = {
+    'jnk': 36, 'DND': 'never', 'LR1': 24, 'LR2': 48, 'LR3': 72, 'LR4': 96,
```

Comment:
even LR1 could have been in lower case? to keep it consistent

Reply:
I pointed this in my comment: https://github.com/red-hat-storage/ocs-ci/pull/1595#discussion_r388837693

I think we should have the lowercases here or we need to do the check for prefixes with lowercase of those keys.

Reply:
I see it, +1 on that or lowercase match to ignore case.

Reply:
The prefix check is done case-insensitive now so functionally it doesn't matter. I figured `LR` was easier to read than `lr`

Reply:
https://github.com/red-hat-storage/ocs-ci/pull/1595/files#diff-3cab31ec2f837c8a4ce76ffd7a784444R88-R89